packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -110,7 +115,10 @@ export const NetworkSelectionContainer = ({
                     >
                       <div className="flex h-6 w-6 items-center justify-center lg:h-6 lg:w-6">
                         <Image
-                          src={getNetworkLogo(Number(chainId))}
+                          src={getNetworkLogo(
+                            Number(chainId),
+                            isLgScreen ? 'dark' : 'light'
```

Comment:
Showing white logo on mobile view, otherwise it blends in with the dark background.